libexec/azhpc-build.sh

```diff
@@ -63,6 +63,9 @@ read_value vnet_name ".vnet.name"
 read_value address_prefix ".vnet.address_prefix"
 read_value admin_user ".admin_user"
 read_value install_node ".install_from"
+if [ ! $(jq -r ".ppg_name" $config_file) = "null" ]; then
```

Comment:
you can just read with a default value here instead of checking with jq

Reply:
I removed the if check and set a default instead. I also change ppg to be more description (proximity_placement_group).